approval_polls/models.py

```diff
@@ -1,3 +1,5 @@
+import logging
```

Comment:
Unused import here. Could you check and fix any style violations in the code you are changing by running the flake8 command as given in the README page (https://github.com/electology/approval_frame#coding-style) ?